packages/terra-framework-docs/src/terra-dev-site/doc/tabs/example/TabsCloseFunc.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import Tabs from 'terra-tabs';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
+import TabContentTemplate from './TabContentTemplate';
+import styles from './common/TabExample.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  id: PropTypes.string,
+};
+
+const TabsCloseFunc = (props) => {
+  const [tabs, setTabs] = useState([]);
+
+  const handleTabClose = (tabdata, itemKey) => {
+    const updatedTabs = tabs.filter((tab) => tab.key !== itemKey);
+    setTabs(updatedTabs);
+  };
+
+  // Healthcare-related tabs
+  const tabsExample = [
+    {
+      label: 'Hospitals',
+      icon: <IconBriefcase />, // Replace with healthcare-related icon
+      isIconOnly: false,
+      key: 'Hospitals',
+      content: 'Find nearby hospitals',
+    },
+    {
+      label: 'Cardiology',
+      icon: <IconSearch />, // Replace with healthcare-related icon
+      isIconOnly: true,
+      key: 'Cardiology',
+      content: 'Cardiology department',
+    },
+    {
+      label: 'Doctors',
+      icon: <i className="fa fa-user-md" />, // Using a generic icon as a placeholder
+      isIconOnly: false,
+      key: 'Doctors',
+      content: 'Search for doctors',
+    },
+    {
+      label: 'Appointments',
+      icon: <i className="fa fa-calendar" />, // Using a generic icon as a placeholder
+      isIconOnly: false,
+      key: 'Appointments',
+      content: 'Manage appointments',
+    },
+    // Add more healthcare-related tabs here
+  ];
+
+  // Initialize the state with the tabsExample data
+  useState(() => {
+    setTabs(tabsExample);
+  }, []);
+
+  return (
+    <div className={cx('content-wrapper')}>
+      <Tabs isClosable onTabClose={handleTabClose} defaultActiveKey="Hospitals" id={props.id}>
+        {tabs.map((tab) => (
+          <Tabs.Pane
+            label={tab.label}
+            isIconOnly={tab.isIconOnly}
+            icon={tab.icon}
+            key={tab.key}
+            id={tab.key}
+          >
+            <TabContentTemplate label={tab.content} id={`TabContent_${tab.key}`} />
+          </Tabs.Pane>
+        ))}
+      </Tabs>
+    </div>
+  );
+};
+
+TabsCloseFunc.propTypes = propTypes;
+export default TabsCloseFunc;
```

Comment:
I do not see this example being imported in the Tabs mdx file. If there are sufficient examples you can remove this file.

Reply:
Right ,This one added while testing and not been deleted, Thanks .
https://github.com/cerner/terra-framework/commit/f4bd3b60b2862061fc633330bda690d094cd0ea6